src/app/cube/about-us/about-us.component.scss

```diff
@@ -0,0 +1,7 @@
+@import '~_vars.scss';
+
+*{
```

Comment:
Can we add 'background-color: white' so that way this is consistent with the theme we have for the details page?

Reply:
Sounds good will do that @EVogel1999